docs/business.yaml

```diff
@@ -4133,15 +4133,15 @@ components:
           type: string
           format: date-time
           example: '2020-11-24T18:23:53.498863+00:00'
-          description: 'The date written on the company''s certificate of incorporation. '
+          description: 'The date written on the company''s certificate of incorporation, amalgamation, Continuation In or registration.'
```

Comment:
Can you help to check with Dylan for this as well? - For consistency, to capitalize the first letter of incorporation/registration, use all lowercase letters for "Continuation In" or just keep it for now?

Reply:
sure

Reply:
Checked with Dylan and updated to "The date written on the company's certificate of Incorporation, Amalgamation, Continuation In or Registration."